Core/Newsletter2Go/Bootstrap.php

```diff
@@ -353,32 +376,25 @@ private function registerEvents()
         $this->subscribeEvent('Enlight_Controller_Front_StartDispatch', 'onEnlightControllerFrontStartDispatch');
         $this->subscribeEvent('Enlight_Controller_Dispatcher_ControllerPath_Backend_Newsletter2go', 'onGetControllerPathBackendNewsletter2go');
         $this->subscribeEvent('Enlight_Controller_Action_PostDispatch_Frontend', 'onFrontendPostDispatch');
+        $this->subscribeEvent(
+          'Enlight_Controller_Front_DispatchLoopStartup',
+          'onStartDispatch'
+        );
     }
 
     /**
      * @throws Exception
      */
     private function registerControllers()
     {
-        // Added to support older versions (<4.2.0)
```

Comment:
this refactoring should still support older shop versions, but it is not needed to support them because the plugin will not get installed because required breaking changes. like the constructors below